### ISSUE TITLE
First underscore with 'connected-underscore' ligation is offset in wezterm

### ISSUE THREAD

**Issue by breucode** [be5invis/Iosevka#1361]
In wezterm, the first underscore of the 'connected-underscore' ligation is offset vertically in wezterm. This does not happen in IntelliJ or VisualStudioCode. However, this issue does not appear with the Fira Code font, so I am not sure if this is an issue with Iosevka or wezterm.

<img width="68" alt="wezterm-iosevka" src="https://user-images.githubusercontent.com/3304638/171119225-f24466dc-282b-4a9c-8813-92a0e68d7d04.png">

<img width="68" alt="wezterm-firacode" src="https://user-images.githubusercontent.com/3304638/171119287-acd59ad6-9aac-46dc-a68c-6b6628725bb4.png">

---

* Your font version: 15.4.2
* Your font variant: SS15, dlig, slab, extended as normal
* Your operating system (name and version): MacOS 12.4 and Ubuntu 22.04
* Your application using Iosevka: wezterm

**Comment by CodexLink** [comment]
Hello @breucode, I have the same issue as yours, but in my case, the misaligned underscore is quite misaligned at the bottom of the `connected underscore` ligature. I noticed that this is too noticeable at the terminal and was not noticeable at any other (GUI) applications, at a certain size point; as per what I stated in (#1389).

Anyway, you might wanna try to **explicitly state** the underscore set to "low" with a section (`buildPlans.xxx.variants.design`) under the variant section (`buildPlans.xxx.variants`) as per what it shows to the second image of the referred issue.
